packages/desktop/components/popup/popups/EvmTransactionFromDappPopup.svelte

```diff
@@ -79,6 +85,26 @@
         }
     }
 
+    async function signOrSend(): Promise<void> {
+        const signedTransaction =
+            method === RpcMethod.EthSendRawTransaction
+                ? rawTransaction
```

Comment:
You're not passing in `rawTransaction` from `eth_transaction.handler`. I get the following error when substituting my tx string in `getEvmTransactionFromHexString`: 


```
Returned error: invalid argument 0: json: cannot unmarshal non-string into Go value of type hexutil.Bytes
```